test/addons-napi/5_function_factory/binding.c

```diff
@@ -16,10 +16,11 @@ napi_value CreateFunction(napi_env env, napi_callback_info info) {
   return fn;
 }
 
-void Init(napi_env env, napi_value exports, napi_value module, void* priv) {
-  napi_property_descriptor desc =
-    DECLARE_NAPI_PROPERTY("exports", CreateFunction);
-  NAPI_CALL_RETURN_VOID(env, napi_define_properties(env, module, 1, &desc));
+napi_value Init(napi_env env) {
+  napi_value exports;
+  NAPI_CALL(env,
+    napi_create_function(env, "exports", CreateFunction, NULL, &exports));
```

Comment:
Nit: Four spaces for continuations, IIRC.

Reply:
Thanks for pointing all these out. Visual Studio strikes again, I guess. I will push an update with these nits fixed. 👍